src/context/auth.jsx

```
@@ -1,31 +1,239 @@
+    });
+
+    // Make auth context available globally for middleware
+    window.authContext = { user: null };
+
+    // Clean up function
+    return () => {
+      unsubscribe();
+      if (syncTimeoutRef.current) {
+        clearTimeout(syncTimeoutRef.current);
+      }
+      delete window.authContext;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)